### ISSUE TITLE
[1.10.2] Iron door texture missing

### ISSUE THREAD

**Issue by Protheus** [Soartex/Issues#10]
Iron door texture missing

![2017-05-01_19 05 13](https://cloud.githubusercontent.com/assets/1455178/25590060/399e43b4-2eaf-11e7-8c70-7c9302d8a180.png)

Using the latest 1.10.2 from curse forge, Iron door texture is missing.

**Comment by Shoeboxam** [comment]
Do you have Malisis doors installed?

**Comment by Protheus** [comment]
Yes i do, so that mod is causing all texture issues ? Or only the iron door one
